internal/formats/common/cyclonedxhelpers/format.go

```diff
@@ -97,18 +105,63 @@ func toBomDescriptor(name, version string, srcMetadata source.Metadata) *cyclone
 	}
 }
 
+func lookupRelationship(ty artifact.RelationshipType) bool {
```

Comment:
nit: the name of this function doesn't really convey what its purpose is, which is to filter down relationship types to those that are meant for package dependencies

Reply:
That's better; what about something like `isExpressiblePackageRelationship`? Beyond naming, I think a helpful comment to help future devs figure which qualities of a relationship indicate that they should be included here.